accelerator/config.c

```diff
@@ -70,7 +83,19 @@ status_t cli_config_set(ta_config_t* const info, iota_config_t* const iconf, ta_
       iconf->seed = value;
       break;
     case CACHE:
-      cache->cache_state = !(strncmp(value, "T", 1));
+      cache->cache_state = (toupper(value[0]) == 'T');
+      break;
+
+    // Verbose configuration
+    case VERBOSE:
+      verbose_mode = (toupper(value[0]) == 'T');
+      break;
+
+    // File configuration
+    case CONF_CLI:
```

Comment:
Maybe rename it to `CONF_FILE`, otherwise it's a bit confusing :smile:

Reply:
For the sake of consistency, I think `CONF_CLI` is ok.